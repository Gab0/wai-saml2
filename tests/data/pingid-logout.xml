<urn:LogoutRequest ID="CVF.JaxEAbpzfJTNB6o3QT92EeN" IssueInstant="2024-02-25T20:47:37.510Z" NotOnOrAfter="2024-02-25T20:52:37.512Z" Version="2.0" Destination="http://localhost:3000/slo/assert" xmlns:urn="urn:oasis:names:tc:SAML:2.0:protocol"><urn1:Issuer xmlns:urn1="urn:oasis:names:tc:SAML:2.0:assertion"/><ds:Signature xmlns:ds="http://www.w3.org/2000/09/xmldsig#"><ds:SignedInfo><ds:CanonicalizationMethod Algorithm="http://www.w3.org/2001/10/xml-exc-c14n#"/><ds:SignatureMethod Algorithm="http://www.w3.org/2001/04/xmldsig-more#rsa-sha256"/><ds:Reference URI="#CVF.JaxEAbpzfJTNB6o3QT92EeN"><ds:Transforms><ds:Transform Algorithm="http://www.w3.org/2000/09/xmldsig#enveloped-signature"/><ds:Transform Algorithm="http://www.w3.org/2001/10/xml-exc-c14n#"/></ds:Transforms><ds:DigestMethod Algorithm="http://www.w3.org/2001/04/xmlenc#sha256"/><ds:DigestValue>aDWirJPyN9SvCaVidvIi//eOPrgUm3CiU2FJpYXk1jE=</ds:DigestValue></ds:Reference></ds:SignedInfo><ds:SignatureValue>EZgKtxxOISEMA8pYqF9vvvwViw7iYyPbQhyH4NiyZHqPt7ZYjSZNVIfoCn1lccDfQGaeFBrWVpds08oYs6Q8LlXzySg3k9VNsghTWvstMXWmLs4/qVAvkzPvL9iCC4tpiQox/kqfgmd1zSJfKtB6CuwgFUtL2gbaSl8/3Oz9jnookCUynw2y7TZHlQ1JexAYB3f9FVUrBhz2vsalk8lSAqrpVpXWRdR2h2p5oQM3nSKJltTwYF+VnyFHRK8PR4594v6nCkb8rgrlKyuzdig5t0NM3f6oriT0+4ZDUonp74xhbJ3qfO5YEmE/Nr3u852VOYB1xH2clsGnz5T1n3Oc6Bc06YiXeQp7/4UQ3GNtshg0Rbxq26D8wm1nVI/Fk62k7er6NLZyvAA4CPqBbnbSx+YFvQnPU3OTYx3XmjWkz0aEHCCSCl1nWfSG4EustcbVkyEUhXSbPCCUWqdVqv4va6rbzZm2B8zxszYH0UyWil49ATJMaWkfQ5OG1fBsvmzI1vrxn11c1SEf/pd1AtQIlcSwiXr9IgAZwCPX1i9eb/9dNro0k61R86EuRPwRax4tMOCfl3HQyLgUCyiRkTEKz1xcL+2jsdIEMELsC6rH6pRYUJoV73vyAmldBO0f9DmVXRAz/6vghggtkGw5dtKlo6iQEaZ/4tcOjMA5nlkwIpQ=</ds:SignatureValue></ds:Signature><saml:NameID Format="urn:oasis:names:tc:SAML:1.1:nameid-format:unspecified" xmlns:saml="urn:oasis:names:tc:SAML:2.0:assertion">tester</saml:NameID><urn:SessionIndex>hopsBnunKnqXmkcgrk00WL6N_n0</urn:SessionIndex><urn:SessionIndex>yd4BI0QGyNsCEJici1UtpqvRJxY</urn:SessionIndex></urn:LogoutRequest> 
