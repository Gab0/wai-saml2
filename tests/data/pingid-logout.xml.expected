AuthnRequest
  { authnRequestTimestamp = 2024-02-25 20:47:37.51 UTC
  , authnRequestID = "CVF.JaxEAbpzfJTNB6o3QT92EeN"
  , authnRequestIssuer = ""
  , authnRequestDestination = Just "http://localhost:3000/slo/assert"
  , authnRequestAllowCreate = False
  , authnRequestNameIDFormat = Unspecified
  , authnRequestNameID =
      Just
        NameID
          { nameIDQualifier = Nothing
          , nameIDSPNameQualifier = Nothing
          , nameIDSPProvidedID = Nothing
          , nameIDFormat = Just Unspecified
          , nameIDValue = "tester"
          }
  , authnRequestSignature =
      Just
        Signature
          { signatureInfo =
              SignedInfo
                { signedInfoCanonicalisationMethod = C14N_EXC_1_0
                , signedInfoSignatureMethod = RSA_SHA256
                , signedInfoReference =
                    Reference
                      { referenceURI = "CVF.JaxEAbpzfJTNB6o3QT92EeN"
                      , referenceDigestMethod = DigestSHA256
                      , referenceDigestValue =
                          "aDWirJPyN9SvCaVidvIi//eOPrgUm3CiU2FJpYXk1jE="
                      }
                }
          , signatureValue =
              "EZgKtxxOISEMA8pYqF9vvvwViw7iYyPbQhyH4NiyZHqPt7ZYjSZNVIfoCn1lccDfQGaeFBrWVpds08oYs6Q8LlXzySg3k9VNsghTWvstMXWmLs4/qVAvkzPvL9iCC4tpiQox/kqfgmd1zSJfKtB6CuwgFUtL2gbaSl8/3Oz9jnookCUynw2y7TZHlQ1JexAYB3f9FVUrBhz2vsalk8lSAqrpVpXWRdR2h2p5oQM3nSKJltTwYF+VnyFHRK8PR4594v6nCkb8rgrlKyuzdig5t0NM3f6oriT0+4ZDUonp74xhbJ3qfO5YEmE/Nr3u852VOYB1xH2clsGnz5T1n3Oc6Bc06YiXeQp7/4UQ3GNtshg0Rbxq26D8wm1nVI/Fk62k7er6NLZyvAA4CPqBbnbSx+YFvQnPU3OTYx3XmjWkz0aEHCCSCl1nWfSG4EustcbVkyEUhXSbPCCUWqdVqv4va6rbzZm2B8zxszYH0UyWil49ATJMaWkfQ5OG1fBsvmzI1vrxn11c1SEf/pd1AtQIlcSwiXr9IgAZwCPX1i9eb/9dNro0k61R86EuRPwRax4tMOCfl3HQyLgUCyiRkTEKz1xcL+2jsdIEMELsC6rH6pRYUJoV73vyAmldBO0f9DmVXRAz/6vghggtkGw5dtKlo6iQEaZ/4tcOjMA5nlkwIpQ="
          , signatureKeyInfo = Nothing
          }
  }